### PR TITLE
Automatically add rpath like linux

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
Probs a good idea to rerender.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -144,3 +144,4 @@ about:
 extra:
   recipe-maintainers:
     - beckermr
+    - isuruf
```
Comment (beckermr) [inline recipe/meta.yaml:147]: +1

**Comment by beckermr** [comment]
There are extra rpath flags that were added to the compiler package tests. We can probably remove those too in a PR on that repo.

**Pull request merged**